rnacentral_pipeline/databases/expressionatlas/helpers.py

```diff
@@ -79,7 +79,7 @@ def as_entry(info, experiment):
         primary_id=primary_id(info),
         accession=accession(info),
         ncbi_tax_id=taxid(info),
-        database="EXPRESSIONATLAS",
+        database="Expression Atlas",
```

Comment:
The database field should be the value we put in `descr` in the `rnc_database` column. This doesn't look like what we generally put, is it?

Reply:
This (the mixed case version) matches what is in the `full_descr` column. I had problems with it in allcaps where the accessions didn't process proprerly I think. I can't remember exactly what the problem was